htdocs/main.inc.php

```diff
@@ -1694,6 +1694,11 @@ function top_httphead($contenttype = 'text/html', $forcenocache = 0)
 			$hookmanager = new HookManager($db);
 		}
 		$hookmanager->initHooks(array("main"));
+		$error = 0;
+		$result = $user->call_trigger('BEFORE_PAGE_LOAD', $user);
```

Comment:
Triggers are reserved for use inside a transaction and on business CRUD event.
To include code here, hooks are a better choice.
If you miss the hook, you can add it, but it must be a hook, not a trigger.

Reply:
> Triggers are reserved for use inside a transaction and on business CRUD event. To include code here, hooks are a better choice. If you miss the hook, you can add it, but it must be a hook, not a trigger.

Thanks for the reply, I'll see about making the changes, but I think a generic hook is missing, I don't think there's a hook that loads regardless of the content.
I'm glad I've learned the difference between hooks and triggers, because I was having a hard time figuring things out.